voluncheer/opportunityboard/tests_views.py

```diff
@@ -77,7 +77,7 @@ def test_update_an_opportunity_page_loads(self):
     def test_search(self):
         """Tests update_an_opportunity page loads"""
         rf = RequestFactory()
-        post_request = rf.post(
```

Comment:
WHy is update_an_opportunity get not post?

Reply:
Since for search we only need to read data, not write, so a get is enough